lib/diagnostic.js

```diff
@@ -13,8 +13,21 @@ db.dropDatabase();
 // House Arryn, motto 'As High as Honor'
 // House Stark, motto 'Winter is Coming'
 // House Targaryen, motto 'Fire and Blood'
-
-// your code begins here
+mongoimport --db=mongo-westeros --collection=houses --type=csv --headerline --file=data/
+db.houses.insert({
```

Comment:
An array of objects should be the first argument if you're trying to insert multiple documents at once.  This is an invalid object.

Here are two solutions.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([{
  name: 'House Arryn',
  motto: 'As High as Honor'
}, {
  name: 'House Stark',
  motto: 'Winter is Coming'
}, {
  name: 'House Targaryen',
  motto: 'Fire and Blood'
}]);
```